cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -1,181 +0,0 @@
-
-The special characters and their meaning are as follows:
-
-.Special characters in syntax summaries
-[options="header", width="100%", cols="1a,3a,3a"]
-|===
-| Character | Meaning | Example
-
-| `\|`
-| Or, used to indicate alternative parts of a command.
-Needs to be part of a grouping.
-| If the syntax needs to specify either a name or `+*+`, this can be indicated with `* \| name`.
-
-| `{` and `}`
-| Used to group parts of the command, common together with `\|`.
-| To use the `or` in the syntax summary it needs to be in a group, `{* \| name}`.
-
-| `[` and `]`
-| Used to indicate an optional part of the command.
-It also groups alternatives together, when there can be either of the alternatives or nothing.
-| If a keyword in the syntax can either be in singular or plural, we can indicate that the `S` is optional with `GRAPH[S]`.
-
-| `...`
-| Repeated pattern, the command part immediately before this is repeated.
-| A comma separated list of names would be `name[, ...]`.
-
-| `"`
-| When a special character is part of the syntax itself, we surround it with `"` to indicate this.
-| To include `{` in the syntax use `"{" { * \| name } "}"`, here we get either `{*}` or `{name}`.
-
-|===
-
-The special characters in the table above are the only ones that need to be escaped using `"` in the syntax summaries.
-
-An example that uses all special characters is granting the `READ` privilege:
-
-[source, cypher, role=noplay]
-----
-GRANT READ
-    "{" { * | property[, ...] } "}"
-    ON {HOME GRAPH | GRAPH[S] { * | name[, ...] }}
-        [
-            ELEMENT[S] { * | label-or-rel-type[, ...] }
-            | NODE[S] { * | label[, ...] }
-            | RELATIONSHIP[S] { * | rel-type[, ...] }
-        ]
-    TO role[, ...]
-----
-
-Some things to notice about this command is that it includes `{` and `}` in the syntax, and between them has a grouping of either a list of properties or the character `*`.
-It also has multiple optional parts, including the entity part of the command which is the grouping following the graph name.
-
-In difference, there is no need to escape any characters in the node property existence constraint creation command.
-This is because `(` and `)` are not special characters, and the `[` and `]` indicate that the constraint name is optional, and are not part of the command.
-
-[source, cypher, role=noplay]
-----
-CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
-ON (n:LabelName)
-ASSERT n.propertyName IS NOT NULL
-----
-
-include::administration-security-users-and-roles.adoc[leveloffset=+1]
-
-[role=enterprise-edition]
-include::administration-security-subgraph.adoc[leveloffset=+1]
-
-[role=enterprise-edition]
-include::administration-security-reads.adoc[leveloffset=+1]
-
-[role=enterprise-edition]
-include::administration-security-writes.adoc[leveloffset=+1]
-
-[role=enterprise-edition]
-include::administration-security-administration.adoc[leveloffset=+1]
```

Comment:
This file is being generated to

neo4j-documentation/4.3/cypher/cypher-docs/target/docs/dev/ql/administration/security/administration-security-administration.adoc

Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/built-in-roles.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/database-administration/

This also contains includedes that should be removed:

cypher/cypher-docs/src/docs/dev/ql/administration/security/

- admin-role-introduction.asciidoc
- database/admin-role-database.asciidoc
- database/admin-database-syntax.asciidoc
- database/grant-database-access-syntax.asciidoc
- database/deny-database-access-syntax.asciidoc
- database/grant-database-start-syntax.asciidoc
- database/deny-database-start-syntax.asciidoc
- database/grant-database-stop-syntax.asciidoc
- database/deny-database-stop-syntax.asciidoc
- database/index-management-syntax.asciidoc
- database/constraint-management-syntax.asciidoc
- database/name-management-syntax.asciidoc
- database/all-management-syntax.asciidoc
- database/transaction-management-syntax.asciidoc
- dbms/admin-role-dbms.asciidoc
- dbms/role-management-syntax.asciidoc
- dbms/user-management-syntax.asciidoc
- dbms/database-management-syntax.asciidoc
- dbms/privilege-management-syntax.asciidoc
- dbms/execute-syntax.asciidoc
- dbms/all-management-syntax.asciidoc